generated/nixnet/nixnet_service.cpp

```diff
@@ -1145,7 +1145,7 @@ namespace nixnet_grpc {
       auto session_ref_grpc_session = request->session_ref();
       nxSessionRef_t session_ref = session_repository_->access_session(session_ref_grpc_session.id(), session_ref_grpc_session.name());
       auto value_buffer = const_cast<f64*>(request->value_buffer().data());
-      u32 size_of_value_buffer = static_cast<u32>(request->value_buffer().size());
+      u32 size_of_value_buffer = static_cast<u32>(request->value_buffer().size()) * sizeof(f64);
```

Comment:
Shouldn't this go inside the `static_cast`? I think this introduces a warning because `sizeof` is a `size_t` and loses precision.

Reply:
Moved `sizeof` inside `static_cast`